changelog/1.19.0.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
```

Comment:
I believe WAC was updated to `0.2` and therefore `0.1` templates no longer work. We may want to call this out. /cc @Emyrk @kylecarbs

Reply:
It needs product documentation too.

Reply:
I've been having issues rebasing/merging master into my PR:https://github.com/cdr/docs/pull/304

Given we are pushing nested templates back a bit, I might trim the `policy` fields for now

Reply:
See text on: https://codercom-67d5emquz-codercom.vercel.app/docs/workspaces/workspaces-as-code/templates

Is that sufficient?

Reply:
@Emyrk I'll take a look at that PR

Reply:
Thanks @khorne3. I was having some issues rebasing and getting the pre-commit stuff to pass. When I merged master in, and couldn't pass the pre-hook, I was unable to commit my merge :smile: